content/03.methods.md

```diff
@@ -117,6 +131,18 @@ We annotated putative driver fusions and prioritized fusions lists with kinases,
 We also added chimerDB [@doi:10.1093/nar/gkw1083] annotations to both driver and prioritized fusion list.
 
 ### Clinical Data Harmonization
+#### WHO Classification of Disease Types
+
+The `disease_type_old` field in the `pbta-histologies.tsv` file contains the diagnosis denoted from the patient's pathology report.
+The `disease_type_new` field in the `pbta-histologies.tsv` file includes updates to `disease_type_old`, including any diagnosis denoted as "Other" as well as a changes on the basis of a molecular alteration.
```

Comment:
I don't quite understand this sentence. Anything that was `Other` in `_old` now gets specified + molecular alterations change things, or something else is meant? This might be better as two sentences.

Reply:
Yeah, hard to explain - we have an internal sheet of diagnoses from path reports plus a column with "other diagnoses" - for some reason, it was determined (not sure when/by who) that certain "other diagnoses" which constitute real diseases (eg: langerhans cell histiocytosis) should be broadly classified as "Other", so the KidsFirst portal shows these as "Other" when in fact a real dx was in the report. I am not sure if this was because they were rarer diseases that they became lumped as other, but I had put the actual diseases back into this file. Maybe I should just say that I used information from the path reports to classify the "Other" diagnoses. Will change this.